script/DeployRouter.s.sol

```diff
@@ -8,14 +8,18 @@ import {Router} from 'src/Router.sol';
 import {DeployBase} from './DeployBase.s.sol';
 
 abstract contract DeployRouter is DeployBase {
+    address public constant SIGNER = 0xffFf5a88840FF1f168E163ACD771DFb292164cFA;
```

Comment:
Would we always use the same signer for multi-chains? If the signer would change, is it better to pass it as a param?

Reply:
Fixed, thanks.